integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -264,22 +264,12 @@ private int getattrInternal(String path, FileStat stat) {
       }
       long size = status.getLength();
       if (!status.isCompleted()) {
-        if (mCreateFileEntries.contains(PATH_INDEX, path)) {
-          // Alluxio master will not update file length until file is completed
-          // get file length from the current output stream
-          CreateFileEntry<FileOutStream> ce = mCreateFileEntries.getFirstByField(PATH_INDEX, path);
-          if (ce != null) {
-            FileOutStream os = ce.getOut();
-            size = os.getBytesWritten();
-          }
-        } else if (!AlluxioFuseUtils.waitForFileCompleted(mFileSystem, uri)) {
```

Comment:
@maobaolong 
Removing the wait will introduce problems in the following workload
1. Fuse write - then getattr() - then possibly read() workload.
Fuse.write() - Fuse.release() which (1)  mCreateFileEntries.remove(ce); (2)  ce.close();
then the new Fuse.getattr() without waiting for file completed will return size = 0, cause the application thinking that the file write failed and error out.
The wait is added to workaround the async release().